IDEHelper/Compiler/BfIRBuilder.cpp

```diff
@@ -1166,11 +1166,7 @@ bool BfIRBuilder::HasExports()
 
 BfIRBuilder::~BfIRBuilder()
 {
-	if (mIRCodeGen != NULL)
-	{
-		mIRCodeGen->mStream = NULL;
```

Comment:
This setting to NULL seemed unnecessary, as this destructor is cleaning up the `mIRCodeGen` pointer entirely and this is a member variable of the class.

Reply:
This `mIRCodeGen->mStream = NULL` was not a no-op. It was to keep the derived classes `BeIRCodeGen` and `BfIRCodeGen` from attempting to delete `mStream` in their destructors since normally they own a stream but in this particular case it was set to read from a stream owned by `BfIRBuilder`.

Reply:
Gotcha. Was trying to think of a case where this would be needed. Thanks for the clarification.